FileProviderExtensionUI/FileProviderCoordinator.swift

```diff
@@ -190,6 +198,15 @@ class FileProviderCoordinator: Coordinator {
 		guard let hostViewController = hostViewController else {
 			return
 		}
+
+		if let fileProviderError = error as? FileProviderCoordinatorError {
+			switch fileProviderError {
+			case let .unauthorized(vaultName):
+				showUnauthorizedError(vaultName: vaultName)
+				return
+			}
+		}
+
```

Comment:
Oh, my suggestion was to put this in `handleError()` at the top (line 64). I believe it fits better there.